webdriver/tests/support/pdf.py

```diff
@@ -4,5 +4,5 @@
 def assert_pdf(value):
     data = decodebytes(value.encode())
 
-    assert data.startswith(b"%PDF-"), "Decoded data starts with the PDF signature"
-    assert data.endswith(b"%%EOF\n"), "Decoded data ends with the EOF flag"
+    assert data.startswith(b"%PDF-"), "Decoded data should start with the PDF signature"
+    assert data.endswith(b"%%EOF"), "Decoded data should end with the EOF flag"
```

Comment:
Please see my investigation on https://bugzilla.mozilla.org/show_bug.cgi?id=1826277#c1. The PDF files as created by Chrome indeed miss a final carriage return. As such this updated assertion is not correct and will break the tests for Firefox.

We could temporarily allow both but then please add a comment with the Chromium bug as reference, and maybe we can have a browser condition here?

Reply:
Added TODO